app-ios/Sources/Strings/Bundle.swift

```diff
@@ -1,28 +0,0 @@
-// Workaround: https://developer.apple.com/forums/thread/664295
```

Comment:
I have removed the whole folder as we no longer need it, let me know what you think.

Reply:
How about Xcode Preview?
This bundle is workaround for Xcode Previews crash.

Reply:
Sorry for not noticing in the previous PR.
Talking about the scope of changes in the last PR, this bundle originally used in Strings was needed to avoid the crash.

Reply:
Anyway, I noticed we should discuss in issues about this 🙏🏼  I'll move this topic to GitHub issue.
Sorry 🙇🏼